lib/exchange/HL7Server.js

```diff
@@ -295,7 +295,7 @@ class HL7Server extends EventEmitter {
         throw new ArgumentError('You must provide HL7Message or string argument');
 
       const str = msg.toHL7();
-      let charset = 'utf8';
+      let charset = this.encoding || 'utf8';
```

Comment:
Note that defaulting to `'utf8'` encoding may be incorrect for HL7, which does not really support variable length character encodings. See https://hl7-definition.caristix.com/v2/HL7v2.7/Tables/0211.

Reply:
@erayhanoglu I noticed you merged this PR already, but please check out this comment.